react-native.config.js

```diff
@@ -4,4 +4,9 @@ module.exports = {
     android: {},
   },
   assets: ['./assets/fonts'],
+  dependencies: {
+    ...(process.env.CI
+      ? {'react-native-flipper': {platforms: {ios: null}}}
```

Comment:
`ios: null` means it's disabled for iOS and enabled on android?

Reply:
Seems like yes, but would be nice to have also on iOS as well.

Reply:
Yeah. As I've understood it is disabled in Android as default.